sct.py

```diff
@@ -320,7 +321,12 @@ def clean_resources(ctx, post_behavior, user, test_id, logdir, dry_run, backend)
             os.environ['SCT_CLUSTER_BACKEND'] = 'aws'
     else:
         os.environ['SCT_CLUSTER_BACKEND'] = backend
-
+    if os.environ.get('SCT_CLUSTER_BACKEND') == 'aws':
+        if regions:
+            regions = regions.split(',')
+        else:
+            regions = SCTConfiguration.aws_supported_regions
```

Comment:
ok, get_all_regions looks better and it's for all the backends. I'll use it.
Btw. anyway possibly I'll need to revert some added logic as otherwise I'd break clean-resources --post-behavior